rails_event_store_active_record/spec/event_repository_spec.rb

```diff
@@ -20,7 +20,7 @@ module RailsEventStoreActiveRecord
     let(:test_race_conditions_auto)  { !ENV['DATABASE_URL'].include?("sqlite") }
     let(:test_race_conditions_any)   { !ENV['DATABASE_URL'].include?("sqlite") }
     let(:test_binary)                { true }
-    let(:test_change)                { !ENV['DATABASE_URL'].include?("sqlite") }
+    let(:test_change)                { true }
```

Comment:
@pawelpacana I  believe ROM also has this built in - not sure if this test is enabled on ROM

Reply:
Right, I'll check this 👍